specification/storage/resource-manager/Microsoft.Storage/stable/2025-01-01/storage.json

```diff
@@ -3088,13 +3091,29 @@
             "name": "DefaultSharePermission",
             "modelAsString": true
           }
+        },
+        "smbOAuthSettings": {
+          "$ref": "#/definitions/SmbOAuthSettings",
+          "description": "Required for Managed Identities access using OAuth over SMB."
         }
       },
       "required": [
         "directoryServiceOptions"
       ],
       "description": "Settings for Azure Files identity based authentication."
     },
+    "SmbOAuthSettings": {
+      "properties": {
+        "isSmbOAuthEnabled": {
+          "type": "boolean",
+          "description": "Specifies if managed identities can access SMB shares using OAuth. The default interpretation is false for this property."
+        }
+      },
+      "required": [
+        "isSmbOAuthEnabled"
+      ],
+      "description": "Setting property for MI access over SMB using OAuth"
```

Comment:
The swagger description will be in rest API doc, and SDK (PSH/CLI) parameter description. 
So please write it clear enough. Like "MI" might can write as the whole word to help customer understand it.